apps/api/src/app/organization/organization.controller.ts

```diff
@@ -154,4 +157,15 @@ export class OrganizationController {
       })
     );
   }
+
+  @Put('/')
```

Comment:
Can we add the member role enum here: 
```
  @Roles(MemberRoleEnum.ADMIN)
```

Reply:
Yes. we want to restrict the update to only members with role `admin`?

Reply:
Yes exactly 🙏

Reply:
@scopsy done